### DISCUSSION TITLE
part-ii/2018-84/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#334] [Question comments]
Part II 2018 Linear Analysis: Paper 2, Section II, F 

https://questions.tripos.org/part-ii/2018-84/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

Essentially all bookwork. 

For (a), it's that the set of compact operators are closed.

For (b), since $K = \overline{T\left(\overline{B}(0,1)\right)}$ is compact, we can apply Arzela-Ascoli to $(C(K),\lVert\cdot\rVert_{\infty})$. 

For (c), assume that $\lambda_j\not\to 0$, and passing to subsequence assume $|\lambda_j|\geq\epsilon\;\forall j$ and $Tx_j\to y$. By Bessel's inequality, $\langle x,y\rangle = 0\;\forall x\in H$, so $y=0$, contradiction.

</details>
